query-engine/js-connectors/js/smoke-test-js/package.json

```
@@ -4,31 +4,34 @@
   "version": "0.0.0",
   "description": "",
   "scripts": {
-    "prisma:db:push:planetscale": "prisma db push --schema ./prisma/mysql-planetscale/schema.prisma --force-reset",
-    "prisma:db:execute:planetscale": "prisma db execute --schema ./prisma/mysql-planetscale/schema.prisma --file ./prisma/mysql-planetscale/commands/type_test/insert.sql",
-    "prisma:planetscale": "pnpm prisma:db:push:planetscale && pnpm prisma:db:execute:planetscale",
-    "prisma:db:push:neon": "prisma db push --schema ./prisma/postgres-neon/schema.prisma --force-reset",
-    "prisma:db:execute:neon": "prisma db execute --schema ./prisma/postgres-neon/schema.prisma --file ./prisma/postgres-neon/commands/type_test/insert.sql",
-    "prisma:neon": "pnpm prisma:db:push:neon && pnpm prisma:db:execute:neon",
-    "prisma:db:push:pg": "prisma db push --schema ./prisma/postgres-pg/schema.prisma --force-reset",
-    "prisma:db:execute:pg": "prisma db execute --schema ./prisma/postgres-pg/schema.prisma --file ./prisma/postgres-pg/commands/type_test/insert.sql",
-    "prisma:pg": "pnpm prisma:db:push:pg && pnpm prisma:db:execute:pg",
-    "neon": "tsx ./src/neon.ts",
-    "pg": "tsx ./src/pg.ts",
-    "planetscale": "tsx ./src/planetscale.ts"
+    "prisma:db:push:neon": "prisma db push --schema ./prisma/postgres/schema.prisma --force-reset",
+    "prisma:db:execute:neon": "prisma db execute --schema ./prisma/postgres/schema.prisma --file ./prisma/postgres/commands/type_test/insert.sql",
+    "prisma:neon": "cross-env-shell DATABASE_URL=\"${JS_NEON_DATABASE_URL}\" \"pnpm prisma:db:push:neon && pnpm prisma:db:execute:neon\"",
+    "neon": "cross-env-shell DATABASE_URL=\"${JS_NEON_DATABASE_URL}\" \"tsx ./src/neon.ts\"",
+
+    "prisma:db:push:pg": "prisma db push --schema ./prisma/postgres/schema.prisma --force-reset",
+    "prisma:db:execute:pg": "prisma db execute --schema ./prisma/postgres/schema.prisma --file ./prisma/postgres/commands/type_test/insert.sql",
+    "prisma:pg": "cross-env-shell DATABASE_URL=\"${JS_PG_DATABASE_URL}\" \"pnpm prisma:db:push:pg && pnpm prisma:db:execute:pg\"",
```

Could we deduplicate & clean this too and make it like this?:
```suggestion
    "prisma:db:push:postgres": "prisma db push --schema ./prisma/postgres/schema.prisma --force-reset",
    "prisma:db:execute:postgres": "prisma db execute --schema ./prisma/postgres/schema.prisma --file ./prisma/postgres/commands/type_test/insert.sql",
    "prisma:postgres": "cross-env-shell DATABASE_URL=\"${JS_PG_DATABASE_URL}\" \"pnpm prisma:db:push:postgres && pnpm prisma:db:execute:postgres\"",
```

---

Thanks for pointing this out. Done.